daemon/network.go

```diff
@@ -113,6 +113,7 @@ func getIpamConfig(data []network.IPAMConfig) ([]*libnetwork.IpamConf, []*libnet
 		iCfg.SubPool = d.IPRange
 		iCfg.Gateway = d.Gateway
 		iCfg.AuxAddresses = d.AuxAddress
+		iCfg.Options = d.Options
```

Comment:
@mavenugo So if we move the ipam options to NetworkCreate we would have to modify the signature of getIpamConfig to pass in the ipam options so libnetwork can do its thing. If we do that we would have to modify CreateNetwork as well to pass in ipamOptions. Is that something everyone is ok with?